lib/attrsets.nix

```diff
@@ -496,6 +496,10 @@ rec {
   getDev = getOutput "dev";
   getMan = getOutput "man";
 
+  /* Get the path to a package's main executable */
+  mainProgram = pkg:
```

Comment:
What do you think `mainBin` to make it a bit shorter? I know it's bike-shedding territory, feel free to ignore.

Reply:
Please let's keep `bin` as the directory. The executable files can be `exe` if we want to keep it short.